packages/any-db-postgres/index.js

```diff
@@ -24,7 +24,7 @@ adapter.createConnection = function(opts, callback) {
 
 inherits(PostgresConnection, pg.Client)
 function PostgresConnection(opts) {
-  pg.Client.call(this, opts)
+  return new pg.Client(opts)
```

Comment:
What problem were you trying to solve here? This would completely break the contract of the library.

Reply:
On new typescript version there is an error class constructor client cannot be invoked without 'new' .

Sure it may break if you use the older version of typescript and target es5 or older. 
But I am just saying that library stopped working on aws Lambda when I updated targeting version in tsconfig and typescipt version.

But anyway I am just using postgresql part of this, so only solution was to change that constructor.

Reply:
I would guess it's a newer version of postgres that has made `pg.Client` a real class.

If your project doesn't depend on any-db features, just replace your imports with imports from pg.

Reply:
Maybe we can use some kind of flag? For example:

```
function PostgresConnection(opts, postgres_version = 7) {
if(postgres_version == 7) {
  pg.Client.call(this, opts);
}
else {
  return new pg.Client(opts);
}
```

And then pass version. It could solve the problem without breaking old versions